tests/unit/test_charm.py

```diff
@@ -455,13 +459,13 @@ def test_buildpack_app_mysql_capability(
                 "BOOT-INF/classes/com/canonical/sampleapp/Application.class",
                 "BOOT-INF/lib/",
                 "BOOT-INF/lib/log4j-to-slf4j-2.19.0.jar",
-            ],
+            ),
             False,
         ),
     ],
 )
 def test_executable_jar_mysql_capability(
-    harness: Harness, patch: SpringBootPatch, files: list[str], expected: bool
+    harness: Harness, patch: SpringBootPatch, files: set[str], expected: bool
```

Comment:
Similar as above, should this be a `tuple`?

Reply:
Fixed